docs/manual/macros.md

```diff
@@ -326,8 +327,8 @@ The macro file syntax is simply:
 %<name>		 <body>
 ```
 
-...where <name> is a legal macro name and <body> is the body of the macro.
-Multiline macros can be defined by shell-like line continuation, ie `\`
+…where `<name>` is a legal macro name and `<body>` is the body of the macro.
+Multiline macros can be defined by shell-like line continuation, ie ``\``
```

Comment:
I'm not sure if this will work, it depends on the renderer. Currently the website is showing \`\\\` un-fenced, implying that it's interpreting the backslash as an escape on the second backtick. This is **unlike** GitHub Flavored Markdown, which properly shows a literal `\`. My change may or may not fix it. The other possibility is that the source should be `` `\\` ``, but in GFM that becomes `\\`.

Reply:
@ffesti may have a better clue. On a related note, we should have docs/README explaining how to test-render locally for rpm.org...

Reply:
^ Another option would be to have GitHub Actions regenerate the website automatically on every push to the `master` branch. (Or, every push that touches the `docs/` subdir, GitHub Actions' job triggers are capable of that level of precision.)

The Action could either export the generated site content as an Artifact (zip file), for manual update of the rpm.org server... or, it could use stored secret credentials to publish directly to the server. Depends on the level of trust and/or immediacy the maintainers are comfortable granting to an automated workflow.

(Arguably, though, any documentation content that's been checked in to the `master` branch is trusted enough that it's implicitly suitable for the live site. If there is an issue, making it live on the site will help expose it quicker so that the bad commit(s) can be reverted or corrected.)

That GitHub Actions workflow file would then serve as (always-correct-and-current) documentation showing **exactly** how the website HTML is generated. `/docs/README` could be as simple as, "Read `/.github/workflows/website.yaml` and run the commands in the `build` step."

Reply:
(I'm assuming the site is pre-rendered to static HTML. If rpm.org runs directly off the `.md` files and renders on demand, then... well, a local static-HTML-rendering job still couldn't hurt to have as part of the CI. And it could be configured to run on PRs that touch the docs, too.)

Reply:
I've created #2259 for the above.